lib/docs/filters/openjdk/entries.rb

```diff
@@ -27,34 +27,17 @@ def get_type
       end
 
       def additional_entries
-        # Only keep the first found entry with a unique name,
-        # i.e. overloaded methods are skipped in index
-        if version == '8' || version == '8 Gui' || version == '8 Web'
-          css('a[name$=".summary"]').each_with_object({}) do |summary, entries|
-            next if summary['name'].include?('nested') || summary['name'].include?('constructor') ||
-                    summary['name'].include?('field') || summary['name'].include?('constant')
-            summary.parent.css('.memberNameLink a').each do |node|
-              name = node.parent.parent.content.strip
-              name.sub! %r{\(.+?\)}m, '()'
-              id = node['href'].remove(%r{.*#})
-              entries[name] ||= ["#{self.name}.#{name}", id]
-            end
-          end.values
+        entries = []
 
-        else
-          css('a[id$=".summary"]').each_with_object({}) do |summary, entries|
-            next if summary['id'].include?('nested') || summary['id'].include?('constructor') ||
-                    summary['id'].include?('field') || summary['id'].include?('constant')
-            summary.parent.css('.memberNameLink a').each do |node|
-              name = node.parent.parent.content.strip
-              name.sub! %r{\(.+?\)}m, '()'
-              id = node['href'].remove(%r{.*#})
-              entries[name] ||= ["#{self.name}.#{name}", id]
-            end
-          end.values
+        css('.memberNameLink a').each do |node|
+          next if  !(node['href'].match?(/\(/)) # skip non-methods
```

Comment:
```suggestion
          next unless node['href'].match?(/[-(]/) # skip non-methods, 8 uses - for (
```

But still, for Java 8 the generated link is http://localhost:9292/openjdk~8/java/lang/stringstring#replace-char-char- (stringstring instead of string) causing a "The page failed to load."